### DISCUSSION TITLE
Introduction of the Romanian language package into the program

### DISCUSSION THREAD

**Discussion by 0xy99** [rejetto/hfs#705] [Ideas]
Hello community! I created a file that includes the menus in Romanian language, a package that I say would help future users of this platform. I'm attaching the file below, if you could implement it as a final file that would be ideal. Thank you for your time!
[hfs-lang-ro.json](https://github.com/user-attachments/files/16596945/hfs-lang-ro.json)

**Comment by rejetto** [comment]
thank you very much for your contribution!
i'll check it and then include it in next release
